src/sidebar/store/modules/selection.js

```diff
@@ -261,6 +280,25 @@ function selectAnnotations(ids) {
   };
 }
 
+/**
+ * Request the UI to give keyboard focus to a given annotation.
+ *
+ * Once the UI has processed this request, it should be cleared with
+ * {@link clearAnnotationFocusRequest}.
+ *
+ * @param {string} id
```

Comment:
Perhaps clarify that this is an id or tag?

Reply:
The value ultimately gets compared against the `thread.id` property, so the answer is actually "must be an ID if the annotation is saved, or a tag otherwise". Currently however the logic in `FrameSyncService` and other selection-related logic only supports saved annotations that have an ID.